action-server/src/threads/worker.ts

```diff
@@ -0,0 +1,38 @@
+import { threadId } from "worker_threads";
+import { jsExecute } from "../utils/codeRunner";
+import { ActionResponse, ActionTask } from "../type/types";
+import { ActionsDbManager } from "../db";
+
+//export async function add({n} : { n : number}) :Promise<number> { debugger;return n; }
+
```

Comment:
We can remove this as this was to test the initial picscina hookup and make sure it was working with a simple function